bridge/Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN yarn build && \
 
 # Use a Docker multi-stage build to create a lean production image.
 # https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds
-FROM node:14-alpine3.15 as production
+FROM node:lts-alpine3.17 as production
```

Comment:
i would prefer to use a pinned node version here. this could have influence on our builds even if we don't change any code since the `lts` tag is a moving one

Reply:
fair point, also for reproducibility of our images